### PR TITLE
[Doc KR] kr/* translated

### PR DESCRIPTION
**Pull request by ChangJoo-Park**
Related issue : #186

### CODE REVIEW

**Comment by ChangJoo-Park** [comment]
@eddyerburgh , Please add @reyoucat to  reviewer. :)

**Comment by eddyerburgh** [comment]
@reyoucat  please could you review this translation?

File: docs/kr/api/components/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# 컴포넌트
+
+vue-test-utils에는 컴포넌트를 스텁하는데 사용할 수 있는 유틸리티 컴포넌트가 포함되어 있습니다.
+
+[TransitionStub](./TransitionStub.md)과 [TransitionGroupStub](./TransitionGroupStub.md)은 기본적으로 `transition` 및 `transition-group` 컴포넌트를 스텁하는데 사용됩니다. 설정을 수정하면 스텁을 수정할 수 있습니다.
```
Comment (marocchino) [inline docs/kr/api/components/README.md]:
```diff
-설정을 수정하면 스텁을 수정할 수 있습니다. 
+설정을 수정해 스텁을 바꿀수 있습니다.
```
or
```diff
-설정을 수정하면 스텁을 수정할 수 있습니다. 
+설정을 수정해 스텁을 변경할 수 있습니다.
```

File: docs/kr/api/config.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+### `stubs`
+
+- 타입: `Object`
+- 기본값: `{
+  transition: TransitionStub,
+  'transition-group': TransitionGroupStub
+}`
+
+컴포넌트에서 사용할 스텁. 마운트 옵션에 전달된 `stubs`가 있으면 덮어쓰입니다.
```
Comment (marocchino) [inline docs/kr/api/config.md]:
```diff
- 마운트 옵션에 전달된 `stubs`가 있으면 덮어쓰입니다.
+ 전달된 마운트 옵션 안에 `stubs`가 있으면 덮어씁니다.
```

File: docs/kr/api/mount.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+- **전달인자:**
+
+  - `{Component} 컴포넌트`
+  - `{Object} 옵션`
+
+- **반환값:** `{Wrapper}`
+
+- **옵션:**
+
+See [options](options.md)
```
Comment (marocchino) [inline docs/kr/api/mount.md]: not translated.

File: docs/kr/api/options.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+})
+
+expect(wrapper.is(Component)).toBe(true)
+```
+
+### `slots`
+
+- 타입: `{ [name: string]: Array<Component>|Component|string }`
+
+컴포넌트에 슬롯 객체를 제공합니다. 키는 슬롯 이름입니다. 이 값은 컴포넌트, 컴포넌트 배열 또는 템플릿 문자열일 수 있습니다.
```
Comment (marocchino) [inline docs/kr/api/options.md]: you can omit last or in Korean.
a, b or c -> a, b, c

File: docs/kr/api/shallow.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  it('renders a div', () => {
+    const wrapper = shallow(Foo, {
+      attachToDocument: true
+    })
+    expect(wrapper.contains('div')).toBe(true)
+  })
+})
+```
+
+**기본, 이름을 가지는 슬롯**
```
Comment (marocchino) [inline docs/kr/api/shallow.md]: 기본 -> 기본값

File: docs/kr/api/wrapper-array/README.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# WrapperArray
+
+`WrapperArray`는 [Wrappers](../wrapper/README.md)의 배열을 가진 객체이고, `Wrapper`를 테스트하는 메소드입니다.
+
+- **속성:**
+
+`length` `number`: `WrapperArray`가 가진 `Wrappers`의 갯수입니다.
```
Comment (marocchino) [inline docs/kr/api/wrapper-array/README.md]: 갯수 -> 개수
ref:
http://www.korean.go.kr/front/mcfaq/mcfaqView.do;front=C548CDB1FB9D36A5DFBB2303CD31CD10?mn_id=62&mcfaq_seq=3096&pageIndex=1

File: docs/kr/guides/choosing-a-test-runner.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
+# 테스트 러너 선택하기
+
+테스트 러너는 테스트를 실행하는 프로그램입니다.
+
+JavaScript 에는 많고 유명한 테스트 러너가 있습니다. `vue-test-utils`는 모든 테스트 러너와 함께 사용할 수 있습니다. 이는 테스트 러너 종류와는 상관없습니다.
```
Comment (marocchino) [inline docs/kr/guides/choosing-a-test-runner.md]:
```diff
- 많고 유명한 테스트 러너가 있습니다
+ 유명한 테스트 러너가 많습니다
```

File: docs/kr/guides/choosing-a-test-runner.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+테스트 러너를 선택할 때는 기능 세트, 성능 및 싱글 파일 컴포넌트, 사전 컴파일 지원 등을 지원하는지 등 몇 가지 사항을 고려해야 합니다. 기존 라이브러리들을 신중하게 비교한 결과, 다음 두 가지 테스트 러너를 권장합니다:
+
+- [Jest](https://facebook.github.io/jest/docs/en/getting-started.html#content)는 완벽한 기능을 갖춘 테스트 러너입니다. 최소한의 설정이 필요하며, 기본적으로 JSDOM을 갖추고 내장된 검증 도구를 제공하며, 커맨드 라인 사용자경험이 뛰어납니다. 그러나 테스트에서 싱글 파일 컴포넌트(SFC: single-file component)를 가져오려면 프리 프로세서(사전 처리 과정)가 필요합니다. 우리는 일반적인 싱글 파일 컴포넌트를 처리할 수 있도록 `vue-jest` 프리프로세서를 만들었습니다. 하지만 vue-loader와 기능적으로 100% 같지는 않습니다.
```
Comment (marocchino) [inline docs/kr/guides/choosing-a-test-runner.md]: 사용자경험 -> 사용성

File: docs/kr/guides/testing-SFCs-with-jest.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+
+``` bash
+npm install --save-dev vue-jest
+```
+
+다음으로 `jest` 블럭을 `package.json`에 추가합니다.:
+
+``` json
+{
+  // ...
```
Comment (marocchino) [inline docs/kr/guides/testing-SFCs-with-jest.md]: invalid json comment.
should be `/* */` style

File: docs/kr/api/components/README.md (hunk 1 above)
Comment (reyoucat) [reply docs/kr/api/components/README.md]: LGTM!
오오 좋아보여요. 위에것으로 하겠습니다.

Comment (ChangJoo-Park) [reply docs/kr/api/components/README.md]: 감사합니다 :)

File: docs/kr/api/shallow.md (hunk 5 above)
Comment (ChangJoo-Park) [reply docs/kr/api/shallow.md]: 감사합니다 엄청 고민했던 부분이었어요

**Comment by reyoucat** [comment]
@eddyerburgh have finished reviewing all the files.

**Comment by eddyerburgh** [comment]
Awesome, great job 🙂

There are some linting errors that need to fixed before we can merge. You can run doc linting: `npm run lint:docs`

**Comment by ChangJoo-Park** [comment]
I'll do linting :)

**Comment by ChangJoo-Park** [comment]
@eddyerburgh Lint is done. @reyoucat is best reviewer I've ever met. additionally, Thanks @marocchino !

@reyoucat , @marocchino 감사합니다 👍

**Comment by eddyerburgh** [comment]
Great work guys, thanks for your hard work 🙏

**Pull request merged**